README.md

```diff
@@ -43,9 +43,35 @@ robmarkcole/deepstack-ui:latest`
 
 Then visit [localhost:8501](http://localhost:8501/) (not whatever ip address is shown in the logs, this is the internal docker ip)
 
+Above examples for using with docker-compose:
+```
+  deepstack:
+    container_name: deepstack
+    restart: unless-stopped
+    image: deepquestai/deepstack:noavx
+    ports:
+      - '5000:5000'
+    environment:
+      - VISION-DETECTION=True
+      - API-KEY="my_key"
+    volumes:
+      - /srv/docker/deepstack:/datastore
+  deepstack_ui:
+    container_name: deepstack_ui
+    restart: unless-stopped
+    image: robmarkcole/deepstack-ui:latest
+    environment: 
+      - DEEPSTACK_IP=192.168.1.133
```

Comment:
There should be a way to pass this in without hard coding?

Reply:
Can you clarify a bit - which part ? 
Environment variables or ... ?

Reply:
The IP address https://docs.docker.com/compose/networking/

Reply:
Ah okay so you mean that you should be able to set a variable in the "deepstack ui" -part of the docker-compose to point at the IP of deepstack without hard coding the IP. 

Right, that can be done by just pointing to the deepstack container, I just tested it works and I'll do the change in the readme.

Reply:
great!